.github/workflows/external-types.toml

```diff
@@ -0,0 +1,17 @@
+allowed_external_types = [
+    "bytes::buf::buf_impl::Buf",
+    "bytes::bytes::Bytes",
+    "http::header",
+    "http::header::map::HeaderMap",
+    "http::method::Method",
+    "http::request::Request",
+    "http::response::Response",
+    "http::status::StatusCode",
+    "http::uri::Uri",
+    "http::version::Version",
+    "http_body::Body",
+    "http_body::frame::Frame",
+    "http_body::size_hint::SizeHint",
+    "tokio::io::async_read::AsyncRead",
+    "tokio::io::async_write::AsyncWrite"
```

Comment:
We've since moved away from publicly depending on Tokio, so these two could be removed.

Reply:
Removed.